docs/reference/indices/field-usage-stats.asciidoc

```diff
@@ -56,6 +56,87 @@ Comma-separated list or wildcard expressions of fields
 to include in the statistics.
 --
 
+[role="child_attributes"]
+[[field-usage-stats-api-response-body]]
+==== {api-response-body-title}
+The response body reports the usage-count of the data structures used to store <<mapping-types, _field data types_>>.
```

Comment:
> I think this API reports the usage per field, rather than per field type? 
This api reports the usage of the data structures that back the fields, not the fields directly.

> it may be better to avoid the word "store", to avoid confusion with stored fields?
Good point!

I like the changes you suggested, thanks!